include/SFML/Window/Event.hpp

```diff
@@ -154,6 +154,39 @@ class SFML_WINDOW_API Event
         Vector2i position; //!< Position of the mouse pointer, relative to the top left of the owner window
     };
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Mouse move raw event
+    ///
+    /// Raw mouse input data comes unprocessed from the
+    /// operating system hence "raw". While the MouseMoved
+    /// position value is dependent on the screen resolution,
+    /// raw data is not. If the physical mouse is moved too
+    /// little to cause the screen cursor to move at least a
+    /// single pixel, no MouseMoved event will be generated. In
+    /// contrast, any movement information generated by the
+    /// mouse independent of its sensor resolution will always
+    /// generate a MouseMovedRaw event.
+    ///
+    /// In addition to screen resolution independence, raw
+    /// mouse data also does not have mouse acceleration or
+    /// smoothing applied to it as MouseMoved does.
+    ///
+    /// Raw mouse movement data is intended for controlling
+    /// non-cursor movement, e.g. controlling the camera
+    /// orientation in a first person view, whereas MouseMoved
+    /// is intended primarily for controlling things related to
+    /// the screen cursor hence the additional processing
+    /// applied to it.
+    ///
+    /// Currently, raw mouse input events will only be generated
+    /// on Windows and Linux.
```

Comment:
I feel like this would require some further highlighting as a hint or whatever doxygen provides.